exercises/tree-building/examples/success-standard/package.yaml

```diff
@@ -0,0 +1,21 @@
+name:                tree-building
+version:             1.0.0.0
```

Comment:
upon further review of other example package.yaml, I find that we just don't have `version` in example `package.yaml`, likely so we don't need to update them in multiple places. So this one should be removed